docs/Fachartikel/Fachartikel.tex

```diff
@@ -170,14 +221,15 @@ \section{Diskussion und Ausblick}
 \item[$-$] Implementierung einer virtuellen Darstellung von Signalen auf einem Oszilloskop.
```

Comment:
Für Computertechnik 2:
was unter anderem Schnittstellen zu externen Geräten wie Speicher enstpricht.

Reply:
Hier bin ich nicht ganz sicher, was Du meinst: Würdest Du den Satz auf der 2. Zeile einfach an den im Text markierten Satz noch dranhängen?

Reply:
ja beim Abschnitt davor bei dem es um CT2 geht, weiter ausführen dass dort vorallem um Anbindung von externen Geräten geht. kann aber auch weggelassen werden

Reply:
Ach so. Ich denke, der Hinweis auf CT2 würde in diesem Kontext genügen. Alles Weitere wäre für mich schon zu sehr ins Detail :)